src/action/macos/create_fstab_entry.rs

```diff
@@ -29,6 +29,9 @@ pub struct CreateFstabEntry {
 }
 
 impl CreateFstabEntry {
+    pub fn typetag() -> &'static str {
```

Comment:
Would it be valuable to make this a part of the `Action` trait, so we can require it be implemented when the action is / leverage the trait system to make it obvious that this method is necessary...? Or would that not make sense / is that already the case without it being a trait method?

Reply:
I originally did this because if you try that, you get this:
```
error[E0038]: the trait `action::Action` cannot be made into an object
   --> src/action/mod.rs:186:1
    |
186 | #[typetag::serde(tag = "action")]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `action::Action` cannot be made into an object
```

Let me play around a little...